src/products/utils/handlers.ts

```diff
@@ -8,6 +8,13 @@ import { toggle } from "@saleor/utils/lists";
 import { ProductAttributeInputData } from "../components/ProductAttributes";
 import { getAttributeInputFromProductType, ProductType } from "./data";
 
+const setPrice = (price: string, initialPrice: number) =>
```

Comment:
I think this could be done with only one condition, like so:
```const setPrice = (price, initialPrice) => !!price ? parseInt(price, 10) : initialPrice```

Reply:
This function is called `setPrice`, but it doesn't actually set anything 🤔